contracts/base/LoansBase.sol

```diff
@@ -280,21 +251,16 @@ contract LoansBase is Base {
         whenLendingPoolNotPaused(address(lendingPool))
         nonReentrant()
     {
-        // TODO Validate allowance amount.
         // calculate the amount of collateral the loan needs in tokens
-        uint256 collateralNeededToken = _getCollateralNeededInTokens(
-            _getTotalOwed(loanID),
-            loans[loanID].loanTerms.collateralRatio
+        (uint256 loanCollateral, , uint256 collateralNeededWei, ) = _getCollateralInfo(
+            loanID
         );
-        uint256 collateralNeededWei = _convertTokenToWei(collateralNeededToken);
 
         // calculate when the loan should end
         uint256 loanEndTime = loans[loanID].loanStartTime.add(
             loans[loanID].loanTerms.duration
         );
 
-        uint256 loanCollateral = loans[loanID].collateral;
-
         // to liquidate it must be undercollateralised, or expired
         require(
             collateralNeededWei > loanCollateral || loanEndTime < now,
```

Comment:
The value of `collateralNeededWei > loanCollateral` is just the variable `requireCollateral` that was calculated by `_getCollateralInfo` on line 255.